From 6278ab7a064eaa53759389c615c6f0ea2ba6e494 Mon Sep 17 00:00:00 2001
From: Benjamin Berg <benjamin.berg@intel.com>
Date: Mon, 25 Dec 2023 12:21:00 +0200
Subject: [PATCH] tests: Write services needed for pyrad

pyrad does an internal call to getaddrinfo with the service "www".
Make sure this service is known by writing an appropriate
/etc/services file.

Signed-off-by: Benjamin Berg <benjamin.berg@intel.com>
---
 tests/hwsim/vm/inside.sh | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tests/hwsim/vm/inside.sh b/tests/hwsim/vm/inside.sh
index d813cf065..38c5284d9 100755
--- a/tests/hwsim/vm/inside.sh
+++ b/tests/hwsim/vm/inside.sh
@@ -93,6 +93,11 @@ tcp     6       TCP
 udp     17      UDP
 ipv6-icmp 58	IPv6-ICMP
 EOF
+# for pyrad
+cat > /etc/services <<EOF
+http            80/tcp          www www-http
+http            80/udp          www www-http
+EOF
 
 # we may need /etc/alternatives, at least on Debian-based systems
 ln -s /tmp/etc/alternatives /etc/
-- 
2.43.0

